README.md

```diff
@@ -282,14 +286,27 @@ The items in `bindings` are instances of `Binding(exchange=.., routingKeyPattern
 ```python
 from tcadmin.resources import Role
 
-hook = Role(
+role = Role(
     roleId=..,
     description=..,
     scopes=(.., ..))
 ```
 
 As with hooks, `scopes` must be a tuple (not a list) of strings.
 
+Roles can be merged if their descriptions match.
```

Comment:
description and roleId?

Reply:
The roleId identifies them for merging.  I guess it could make sense phrased either way..